deployments/ansible/roles/collector/README.md

```diff
@@ -105,8 +126,40 @@ how to use the role in a playbook with minimal required configuration:
 - `splunk_fluentd_config`: Path to the fluentd config file on the remote host.
   (**default:** `/etc/otel/collector/fluentd/fluent.conf`)
 
-- `splunk_fluentd_config_source`: Source path to a fluentd config file on your 
```

Comment:
This is still an active variable? Why is the description being removed?

Reply:
Added back.